src/utils/dataDisposal.utils.ts

```diff
@@ -0,0 +1,51 @@
+import { APIRequestContext } from '@playwright/test';
+import { CustomersApiService } from 'api/services/customers.api-service';
+import { ProductsApiService } from 'api/services/product.api-service';
+import { OrdersAPIService } from 'api/services/orders.api-service';
+
+export class DataDisposalUtils {
+  private ordersApiService: OrdersAPIService;
+  private customersApiService: CustomersApiService;
+  private productsApiService: ProductsApiService;
+
+  constructor(context: APIRequestContext) {
+    this.ordersApiService = new OrdersAPIService(context);
+    this.productsApiService = new ProductsApiService(context);
+    this.customersApiService = new CustomersApiService(context);
+  }
+
+  async clearOrders(orderId: string, token: string) {
+    if (!orderId) {
+      return;
+    }
+    try {
+      await this.ordersApiService.deleteOrder(orderId, token);
+    } catch (error) {
+      console.error(` The order ${orderId} was not deleted}`, error);
+    }
+  }
+
+  async clearProducts(productsIds: string[], token: string) {
+    if (!productsIds || productsIds.length === 0) {
+      return;
+    }
+    for (const productId of productsIds) {
+      try {
+        await this.productsApiService.delete(productId, token);
+      } catch (error) {
+        console.error(` The product ${productId} was not deleted}`, error);
+      }
+    }
+  }
```

Comment:
Токен можем не передавать, нам не важно в данном случае, как или кем удаляются данные, это просто чистка
```suggestion
  private token = '';
  
  private async prepareToken(): Promise<string> {
    if (!this.token) {
      this.token = await signInService.login();
    }
    return this.token;
  }

  private async getToken(token?: string): Promise<string> {
    return token ?? await this.prepareToken();
  }
  
  async clearProducts(productsIds: string[]) {
    if (!productsIds.length) return;
    const authToken  = await this.getToken();
    
    for (const productId of productsIds) {
      try {
        await this.productsApiService.delete(productId, authToken);
      } catch (error) {
        console.error(` The product ${productId} was not deleted}`, error);
      }
    }
  }
```